frontend/src/pages/index.js

```diff
@@ -11,7 +10,10 @@ const IndexPage = () => (
   <Layout>
     <div className="w-full">
       <div className="relative text-center text-Shades-0">
-        <img className="w-full h-full mb-0" src={homeImage} />
+        <StaticImage
+          className="w-full h-full mb-0"
+          src="../images/image245.png"
+        />
```

Comment:
It seems image is a bit cut off.  
website: 
![image](https://github.com/hmcc-global/hmccaa-web/assets/9808478/67b2dd15-1f19-4d9e-bc86-e2bfad2ba6da)

Actual image: 

![image](https://github.com/hmcc-global/hmccaa-web/assets/9808478/51544757-39fe-4fb8-a774-d3b7e5ff0bb4)